10.0/apache/docker-entrypoint.sh

```diff
@@ -21,25 +27,13 @@ if version_greater "$image_version" "$installed_version"; then
     if [ "$installed_version" != "0.0.0~unknown" ]; then
         su - www-data -s /bin/bash -c 'php /var/www/html/occ app:list' > /tmp/list_before
     fi
-
     rsync -a --delete --exclude /config/ --exclude /data/ --exclude /custom_apps/ --exclude /themes/ /usr/src/nextcloud/ /var/www/html/
-
-    if [ ! -d /var/www/html/config ]; then
-        cp -arT /usr/src/nextcloud/config /var/www/html/config
-    fi
-
-    if [ ! -d /var/www/html/data ]; then
-        cp -arT /usr/src/nextcloud/data /var/www/html/data
-    fi
-
-    if [ ! -d /var/www/html/custom_apps ]; then
-        cp -arT /usr/src/nextcloud/custom_apps /var/www/html/custom_apps
-        cp -a /usr/src/nextcloud/config/apps.config.php /var/www/html/config/apps.config.php
```

Comment:
This pull request breaks this line: if /var/www/html/config exists but /var/www/html/custom_apps doesn't, config/apps.config.php don't get copied